woocommerce/woocommerce
# Blocks: Add missing `wp-block-woocommerce-{name}` to Add to Cart with Options, Product Image, Product Rating, Product Rating Stars, Product Rating Counter and Product Image

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Part of #49739 .

This PR adds the missing `wp-block-woocommerce-{name}` to the following blocks:

- Product Image
- Product Rating
- Product Rating Stars
- Product Rating Counter
- Add to Cart with Options



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Visit the shop page (/shop) and click on a product.
2. After being redirected to the Single Product page, make sure the Product Image, Add to Cart with Options, Product Rating, Product Rating Counter and Product Ratings Stars blocks are being rendered correctly.
    2.1. To test the Product Rating Stars and Product Rating Counter, follow these instructions:
    2.2. Go to a template/post/page;
    2.3. Add the Single Product block. 
    2.4. Switch to the "Code Editor" mode (in MacOS: Shift + Option + CMD + M);
    2.5. Add the following within the Single Product block:

         ```
          

          
          ```

5. Save and check the changes on the frontend.
3. Inspect each block and make sure you can see the `wp-block-woocommerce-{block-name}` attached to the class property of the HTML tag. (`{block_name}` is the name of the block, for instance for Product Rating, it is `product-rating`). 



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
Add missing `wp-block-woocommerce-{name}` class to Add to Cart with Options, Product Image, Product Rating, Product Rating Stars, Product Rating Counter and Product Image blocks.
</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
